### PR TITLE
PAYARA-2423 Allow Adding an Instance to more than one Deployment Group upon Creation.

### PR DESCRIPTION
**Pull request by MeroRai**

### CODE REVIEW

**Comment by MeroRai** [comment]
Jenkins, quick test Payara 5 please

File: nucleus/cluster/admin/src/main/java/fish/payara/admin/cluster/ListDeploymentGroupsCommand.java [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+import org.glassfish.hk2.api.PerLookup;
+import org.jvnet.hk2.annotations.Service;
+
+/**
+ * Command to list Deployment Groups
+ *
+ * @since 5.0
+ * @author Susan Rai
+ */
+@Service(name = "list-deployment-group")
```
Comment (Pandrex247) [inline nucleus/cluster/admin/src/main/java/fish/payara/admin/cluster/ListDeploymentGroupsCommand.java]: Surely `list-deployment-groups`?

File: nucleus/cluster/admin/src/main/java/fish/payara/admin/cluster/ListDeploymentGroupsCommand.java [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+            Properties extrasProps = new Properties();
+            ArrayList<String> deploymentGroupNames = new ArrayList<String>();
+            for (DeploymentGroup deploymentGroup : listOfDeploymentGroup) {
+                sb.append("\t" + deploymentGroup.getName() + "\n");
+                deploymentGroupNames.add(deploymentGroup.getName());
+            }
+            extrasProps.put("listOfDeploymentGroups", deploymentGroupNames);
+            report.setMessage(sb.toString());
+            report.setExtraProperties(extrasProps);
+            report.setActionExitCode(ActionReport.ExitCode.SUCCESS);
```
Comment (Pandrex247) [inline nucleus/cluster/admin/src/main/java/fish/payara/admin/cluster/ListDeploymentGroupsCommand.java]: This is unnecessary - you only need to set it to `SUCCESS` if it's been set to `FAILURE` somewhere else.

**Comment by MeroRai** [comment]
Jenkins, quick test Payara 5 please

**Pull request merged**